src/pages/api/gitmojis/index.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import gitmojisData from 'src/data/gitmojis.json'
+
+const gitmojis = (request: { method: string }, response: Object): void => {
+  const { method } = request
+
+  if (method === 'GET') {
+    response.status(200).json(gitmojisData)
+    return
+  }
+
+  response.setHeader('Allow', ['GET'])
+  response.status(405).end(`Error: method ${method} not allowed`)
```

Comment:
Not really important but maybe it's more common to see a json object for an errored response:
```suggestion
  response.status(405).json({error: `method ${method} not allowed`})
```

Reply:
Yeah sure! Great point 👍🏼